listenbrainz/webserver/static/css/main.less

```
@@ -62,7 +62,7 @@ body {
   .navbar-form {
     padding-right: 0px;
     .btn {
-      background-color: @turquoise;
+      background-color: #eb743b;
```

```suggestion
      background-color: @orange;
```
See this file: https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/webserver/static/css/theme/variables.less#L11-L22

---

Thanks for the correct implementation there. I missed correcting that during development and was testing variations on the spot. Hah!